libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/plugin/SalesforceOAuthPlugin.java

```diff
@@ -105,15 +105,8 @@ protected void authenticate(final CallbackContext callbackContext) throws JSONEx
      * @throws JSONException
      */
     protected void getAuthCredentials(CallbackContext callbackContext) throws JSONException {
-        Log.i("SalesforceOAuthPlugin.getAuthCredentials", "getAuthCredentials called");
-    	JSONObject credentials = ((SalesforceDroidGapActivity) cordova.getActivity()).getJSONCredentials();
-        if (credentials == null) {
-            Log.w("SalesforceOAuthPlugin.getAuthCredentials", "getAuthCredentials failed - never authenticated");
-            callbackContext.error("Never authenticated");
-        } else {
-            Log.i("SalesforceOAuthPlugin.getAuthCredentials", "getAuthCredentials successful");
-            callbackContext.success(credentials);
-        }    
+        Log.i(TAG, "getAuthCredentials called");
```

Comment:
Refactored to keep the cordova plugin and the react bridge as similar as possible.